llvm/test/CodeGen/PowerPC/gcov_ctr_ref_init.ll

```diff
@@ -0,0 +1,129 @@
+; Tests if the __llvm_gcov_ctr section contains a .ref pseudo-op
+; referring to the __llvm_covinit section.
+; RUN: llc < %s | FileCheck --check-prefixes=CHECK,CHECK-RW %s
+; RUN: llc -mxcoff-roptr < %s | FileCheck --check-prefixes=CHECK,CHECK-RO %s
+
+target datalayout = "E-m:a-p:32:32-Fi32-i64:64-n32"
+target triple = "powerpc-ibm-aix"
+
+; CHECK-RW: .csect __llvm_covinit[RW],3
+; CHECK-RO: .csect __llvm_covinit[RO],3
+; CHECK:         .vbyte  4, __llvm_gcov_writeout[DS]
+; CHECK-NEXT:    .vbyte  4, __llvm_gcov_reset[DS]
```

Comment:
It is important that no active-csect changing directives are present. I suggest just checking the two lines in the middle in a block of `CHECK-NEXT`s.
```
.align 3                               # @__llvm_covinit_functions
L..__llvm_covinit_functions:
.vbyte 4, __llvm_gcov_writeout[DS]
.vbyte 4, __llvm_gcov_reset[DS]
```

Reply:
The test is modified to make sure we use `CHECK-NEXT`s after the csect changing directive. 

> It is important that no active-csect changing directives are present.

Could you help me understand why this is the case? The intention of the test is to make sure that the `__llvm_covinit` csect has the correct storage mapping class and contents. That's why the test includes the `csect` directive.

Reply:
> Could you help me understand why this is the case? The intention of the test is to make sure that the `__llvm_covinit` csect has the correct storage mapping class and contents. That's why the test includes the `csect` directive.

I meant it is important that we guard against active-csect changing directives in the middle. Meaning that my suggestion was to avoid skipping the two lines in the middle.

Reply:
Oh I see - yeah I think with the updated test I am on the same page with you. We do not skip any lines after the first `csect` changing directive now. 

Thanks for the clarification!